code/modules/shuttle/ftl.dm

```diff
@@ -48,6 +48,59 @@
 	width = SSmapping.config.ftl_ship_width
 	height = SSmapping.config.ftl_ship_height
 
+/obj/docking_port/mobile/fob
+	name = "FTL FOB"
+	id = "fob"
+	callTime = 650
+	preferred_direction = EAST
+	area_type = /area/shuttle/ftl/cargo/mining
+	var/previous_dock
+
+/obj/docking_port/mobile/fob/Initialize(mapload)
+	dir = SSmapping.config.fob_shuttle_dir
+	dwidth = SSmapping.config.fob_shuttle_dwidth
+	dheight = SSmapping.config.fob_shuttle_dheight
+	width = SSmapping.config.fob_shuttle_width
+	height = SSmapping.config.fob_shuttle_height
+	. = ..()
```

Comment:
This is the first time I actually looked at this bit of code, make `/obj/docking_port/mobile/fob` the parent of everything else so they share an Initialize()